shared/menubar/index.desktop.js

```diff
@@ -43,9 +43,9 @@ class MenubarRender extends Component<Props, State> {
 
   constructor(props: Props) {
     super(props)
-    SafeElectron.getRemote()
-      .getCurrentWindow()
-      .on('show', this._onShow)
+    if (SafeElectron.getCurrentWindowFromRemote) {
```

Comment:
I was getting errors about SafeElectron not having a `getRemote` or `getCurrentWindow` property when I ran this in storybook. I think the reason why is in [`__mocks__/electron.js`](https://github.com/keybase/client/blob/637af207887e94ee23e916bf76b485f047225e74/shared/__mocks__/electron.js#L8), where `remote` is initialized to an almost empty object.

I also noticed that there was a single function (probably written after this code was last touched?) called [`getCurrentWindowFromRemote`](https://github.com/keybase/client/blob/637af207887e94ee23e916bf76b485f047225e74/shared/util/safe-electron.desktop.js#L83) that basically wrapped around these two functions. The cleanest solution seemed like to just check for this function before using it, but I could see this potentially not throwing an exception somewhere when we would want it to?